test/functional/cmdline_options_testresources/src_90/com/ibm/j9/getcallerclass/RefectionMHTests.java

```diff
@@ -35,8 +35,9 @@
 public class RefectionMHTests {
 
 	private static boolean isSecurityFrameOrInjectedInvoker(Class<?> cls) {
+		String injectedInvoker = (VersionCheck.major() < 15) ? ".InjectedInvoker/" : "$$InjectedInvoker/";
 		return ("java.lang.invoke.SecurityFrame" == cls.getName()
-			|| cls.getName().startsWith(RefectionMHTests.class.getName() + "$$InjectedInvoker/"));
+			|| cls.getName().startsWith(RefectionMHTests.class.getName() + injectedInvoker));
```

Comment:
During the rewrite, I forgot to account for `RefectionMHTests.class.getPackageName()` in `VersionCheck.major() < 15`.

Reply:
Yes. Can't believe I overlooked that as well. What do you think, a multi-line ternary, or the previous `if`-statement approach?

While we are here, could we fix these test names? Clearly it was meant to be `ReflectionMHTests` rather than `RefectionMHTests`.

Reply:
Yes, we should fix the typo in the test name.

Reply:
Is the result of the below code the same?
```
boolean rc = ("java.lang.invoke.SecurityFrame" == cls.getName());
vs
boolean rc = "java.lang.invoke.SecurityFrame" == cls.getName();
```

Reply:
Yes, `==` has higher precedence. Though, I can change it if you'd like.

Reply:
No, it's fine; just wanted to confirm.